test/SparqlParserTest.cpp

```diff
@@ -1179,3 +1180,95 @@ TEST(ParserTest, Bind) {
   ASSERT_EQ(bind._target, "?a");
   ASSERT_EQ(bind._expression.getDescriptor(), "10-5");
 }
+
+TEST(ParserTest, Order) {
+  {
+    ParsedQuery pq =
+        SparqlParser("SELECT ?x ?y WHERE { ?x :myrel ?y }").parse();
+    ASSERT_TRUE(pq._orderBy.empty());
+    ASSERT_EQ(pq._rootGraphPattern._children.size(), 1);
+    ASSERT_TRUE(holds_alternative<GraphPatternOperation::BasicGraphPattern>(
+        pq._rootGraphPattern._children[0].variant_));
```

Comment:
No need to test them here, we have many tests for these, you can just check the OrderBy clause.